doc/user/sql/create-sink.md

```diff
@@ -26,22 +26,28 @@ Field | Use
 **IF NOT EXISTS** | If specified, _do not_ generate an error if a sink of the same name already exists. <br/><br/>If _not_ specified, throw an error if a sink of the same name already exists. _(Default)_
 _sink&lowbar;name_ | A name for the sink. This name is only used within Materialize.
 _item&lowbar;name_ | The name of the source or view you want to send to the sink.
+**FILE** _path-prefix_ | The absolute path and file name prefix of file to create and write to.
```

Comment:
nit: I realize that this is annoying, but can you amend the variable names to use underscores instead of hyphens to make them consistent with the other variable names? Glad to go back through and do this for you if it's too obnoxious.